### DISCUSSION TITLE
Changing Mounted Volume Permissions

### DISCUSSION THREAD

**Discussion by BenA-SA** [cookiecutter/cookiecutter-django#3973] [Q&A]
Hi All,

I have added Selenium to my Cookiecutter-Django made Docker environment as follows:
`  selenium:
    image: selenium/standalone-chrome-debug:3.141.59-20210105
    volumes:
      - .:/app
      - /app/sel_downloads:/home/seluser/Downloads`

I am trying to use selenium to download files into the mounted volume (home/seluser/Downloads) and then access these files within django (app/sel_downloads). However, when attempting to download to this file, I get a permission denied error.

According to [this solution](https://github.com/SeleniumHQ/docker-selenium#mounting-volumes-to-retrieve-downloaded-files), I need to create the directory and change it's permissions - I have tried this but still seem to be getting the same error. Does anyone know how to change the permissions of this mounted volume as part of the Cookiecutter-Django docker build process?

Many thanks for any help anyone can offer!

**Comment by exotriks** [comment]
probably somewhere in the mountpoints if you want to change permissions of the volume during the build process
